backend/typescript/graphql/types/staffType.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from "apollo-server-express"
+
+const staffType = gql `
+  type StaffDTO {
+    id: ID!
+    roleId: Float!
```

Comment:
Are role IDs expected to be a Float? If a decimal place in roleID isn't expected maybe this should be changed to Int/String (if they're UUIDs)